bucket/perfview.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "3.0.1",
+    "description": "PerfView is a CPU and memory performance-analysis tool",
+    "homepage": "https://github.com/Microsoft/perfview",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/microsoft/perfview/releases/download/v3.0.1/PerfView.exe",
+            "hash": "2e00ad19de6abc5f776f5a3539748962778b37db0472913a54b8eb0ca06a2286"
+        }
+    },
+    "bin": "PerfView.exe",
```

Comment:
Does the binary support commandline arguments?

Reply:
yes, it can even be started headless/without GUI